tests/tests/in-cluster-eventing.spec.js

```diff
@@ -43,7 +43,7 @@ context('In-cluster eventing', () => {
 
     cy.getIframeBody()
       .find(
-        '[placeholder="Enter Event type, for example, sap.kyma.custom.test-app.order.cancelled.v1"]',
+        '[placeholder="Enter event type, for example, sap.kyma.custom.test-app.order.cancelled.v1"]',
```

Comment:
I can't add a suggestion here, but don't you need `the` here? 
You have it [here](https://github.com/kyma-project/busola/blob/cae15a57d5f5a51b5f55d730d9e3419356e46555/core/src/i18n/en.yaml#L1640).

Reply:
Of course I do, I fixed it one place and forgot about this one, good catch! 🙂

Reply:
Perfect, thank you! 😊